### DISCUSSION TITLE
Persistence control in ElsaV3

### DISCUSSION THREAD

**Discussion by Invenisso** [elsa-workflows/elsa-core#3532] [Design & tech]
Hello Community, hello Sky Walker! :-)

In Elsa V2 we can choose when workflow will be persisted:

![image](https://user-images.githubusercontent.com/57659054/207410076-e030bd14-44e1-40cf-88c9-e9fd9f12ca98.png)

In current pre-release version of Elsa V3 workflow is persisted only when finished or suspended (at the bookmark). Will there also be an option to persist after activity execution in ElsaV3? I would like to resume workflow from where it left off in case of a sudden shutdown of the workflow server.

**Comment by sfmskywalker** [comment]
@Invenisso Although this is currently not implemented. I agree that this is an important feature to bring over. I'll create an issue for it. Thanks for the question!

**Reply by sfmskywalker** [reply]
Issue created: #3603

**Comment by Invenisso** [comment]
Thank you for your answer and creating the issue. I hope that the work on Elsa V3 goes smoothly. :-)
